src/confetti.js

```diff
@@ -281,18 +281,11 @@
     return colors.map(hexToRgb);
```

Comment:
```suggestion
    return colors.map(colorToRgb);
```

Naming: the function no longer only accepts hex strings.